### DISCUSSION TITLE
prevent production builds from minifying

### DISCUSSION THREAD

**Discussion by the-mercenaries-ltd** [vercel/community#135] [Help] [answered]
for some reason my production build: https://www.zeropigsgiven.com/ is minifying my code and now typeof is not working in some instances because none of my objects/classes/instances have proper types

i tried creating a next.config..js with 

```module.exports = {
  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
    config.optimization.minimize = false;
    return config
  },
}
``` but it's still being minified , any suggstions?

**Comment by MathiasReker** [comment]
Can you send the full build pipeline?

**Comment by Pieparker** [comment, marked as answer]
This conversation appears to have gone quiet. To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
